avm/ptn/policy-insights/remediation/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,152 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
```

Comment:
Please update metadata name and description to waf-aligned related tests (you can reference any res module)

Reply:
The description is still to be updated. Also please do not resolve comments yourself, it makes the review more difficult 😉

Reply:
Changed to:
metadata name = 'WAF-aligned'
metadata description = 'This instance deploys the module in alignment with the best-practices of the Azure Well-Architected Framework.'

Reply:
Thank you! Please also rerun the Set-AVMModule utility for changes to be reflected in the Readme as well

Reply:
Updated readme